qa/setup_packages.py

```diff
@@ -177,8 +230,11 @@ def main():
     elif args.getall:
         print(get_all_strings(args.use, args.cuda))
     elif args.install >= 0:
-        if args.install > cal_num_of_configs(args.use, args.cuda):
+        if args.install > cal_num_of_configs(args.use, args.cuda) or args.install < 0:
```

Comment:
We are in branch where args.install >= 0, it can't ever satisfy the `or args.install < 0` clause.

Reply:
Done